hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1912,6 +1918,8 @@ private void deleteOnConsumedBlobs(final Path srcPath,
     } finally {
       deleteBlobExecutorService.shutdown();
     }
+
+    tracingContext.setOperatedBlobCount(deletedBlobCount.get() + 1);
```

Comment:
Same here ?

Reply:
Increment is being done only for blobs in renamed in parallel

This +1 is for blob which would contain the count value in CID.